chartfx-samples/src/main/java/de/gsi/dataset/samples/DataSetToByteArraySample.java

```diff
@@ -102,7 +102,11 @@ public void testGenericSerialiserIdentity(final boolean asFloat32) {
         byteBuffer.reset(); // '0' writing at start of buffer
         serialiser.serialiseObject(dsOrig);
         byteBuffer.reset(); // reset to read position (==0)
-        serialiser.deserialiseObject(cpOrig);
+        final Object retOrig = serialiser.deserialiseObject(cpOrig);
+
+       if (cpOrig != retOrig) {
```

Comment:
Codacy found an issue: [Use equals() to compare object references.](https://app.codacy.com/gh/GSI-CS-CO/chart-fx/pullRequest?prid=6382991)